### DISCUSSION TITLE
An option to disable the gyro effect on the Live Wallpaper would be nice

### DISCUSSION THREAD

**Discussion by Rugryder** [breezy-weather/breezy-weather#342] [Ideas]
Found out about this app from it being posted on Reddit and it's been pretty fantastic so far. It would be nice though if we could have an option to disable the gyro effect for the live wallpaper. Just to help save a little bit of battery.

**Comment by papjul** [comment]
The sensor option from "Main screen" settings also applies to live wallpaper. If not, please open an issue.

**Reply by Rugryder** [reply]
I did see that and it does work, but having it as an option for just the live wallpaper would be nice. I actually like the effect when in the app.
